src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -233,7 +233,7 @@
   <data name="CannotOpenFile" xml:space="preserve">
```

Comment:
Hi @junmendoza, just want to make sure you modified `Resources.resx`, then overwrite `Resources.en-US.resx` with `Resources.resx`? Just trying to ensure these files are completely identical.